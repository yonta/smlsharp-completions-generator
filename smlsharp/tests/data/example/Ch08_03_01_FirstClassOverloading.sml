1 + 1;
1.0 + 1.0;
0w1 + 0w1;

fun plus x = x + x;
