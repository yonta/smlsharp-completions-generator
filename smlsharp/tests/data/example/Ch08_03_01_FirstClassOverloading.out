val it = 2 : int

val it = 2.0 : real

val it = 0wx2 : word

val plus = fn : ['a::{int, word, int8, word8, ...}. 'a -> 'a]
